pkgs/tools/audio/alsaequal/alsaequal.1

```diff
@@ -0,0 +1,146 @@
+.TH ALSAEQUAL 1
```

Comment:
So did you just wrote a man page for this?

Reply:
Yes. I decided not to use "etc/asound.conf" as it could cause collision with other packages, so I basically rewrote "README" that states what each option in configuration is for and added a configuration example suitable for nix.

Reply:
This is not considered a good place for such usage documentation, especially when it's meant specifically for NixOS. I'd add documentation to what shows up eventually in https://nixos.org/nixpkgs/manual/#chap-packages , i.e: https://github.com/NixOS/nixpkgs/tree/master/doc/builders/packages/ .

Reply:
That is not a usage documentation, it describes internal plugin options that can be changed in configuration. The only "usage" part is an example of a configuration, that is suitable for nix.

Reply:
Well if it's not specific to NixOS then it should be provided by upstream from a brief read it's seem to be available in their website. If I'm wrong, maybe try suggest it to them... As for the NixOS specific stuff, that should definitely go to the Pacakges' notes.

Reply:
Well, it's a shame then. To use this plugin on nix you have to point the configuration to the right library, but the way to do so is not particularly widespread, and I haven't been able to find it in the ALSA documentation, so the end-user will be left on his own. But I do understand your concerns, so I will just push the upstream version (likely tomorrow, I have some technical difficulties currently), and maybe if I find some time later, I will add this info to nixos packages notes.

Reply:
You are more then invited to add as much information as you feel necessary to NixOS' packages notes. I'm only asking you to change the format (from a man page to a worse format) and put it in the right place, not in the package it self.